python-libraries/data-platform-catalogue/pyproject.toml

```diff
@@ -9,7 +9,9 @@ packages = [{ include = "data_platform_catalogue" }]
 
 [tool.poetry.dependencies]
 python = "^3.10.0"
-openmetadata-ingestion = "^1.1.7.0"
+# pinned to this version as was installing a later vesion and giving errors, seems a breaking
+# change introduced inadvertently
+openmetadata-ingestion = "1.1.7.0"
```

Comment:
annoying... seems like the library is changing quite fast at the moment though so hopefully the problem goes away